NuGet.config

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <solution>
+    <add key="disableSourceControlIntegration" value="True" />
+  </solution>
+  <packageRestore>
+    <add key="enabled" value="True" />
+    <add key="automatic" value="True" />
+  </packageRestore>
```

Comment:
I don't think these are needed, particularly not when building with `dotnet build`. Seems like legacy stuff. All my VS and CI stuff have always worked without these.